### PR TITLE
Add suffixes to typos from "ta" to "th"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -51388,6 +51451,12 @@ tennisplayer->tennis player
 tentaive->tentative
 tentaively->tentatively
 tention->tension
+teorem->theorem
+teorems->theorems
+teoretical->theoretical
+teoretically->theoretically
+teories->theories
+teory->theory
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:51459]: Also `teary`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -51388,6 +51451,12 @@ tennisplayer->tennis player
 tentaive->tentative
 tentaively->tentatively
 tention->tension
+teorem->theorem
+teorems->theorems
+teoretical->theoretical
+teoretically->theoretically
+teories->theories
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:51458]: Perhaps `Tories` too (in which case add singular below)?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -51417,9 +51517,31 @@ terminats->terminates
...
+terminiate->terminate
+terminiated->terminated
+terminiates->terminates
+terminiating->terminating
+terminiation->termination
+terminiations->terminations
 terminiator->terminator
-terminte->terminate
+terminiators->terminators
+terminl->terminal
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:51531]: Perhaps `termini` (plural of `terminus`)?

Comment (int-y1) [reply codespell_lib/data/dictionary.txt:51531]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**